batch/batch/front_end/front_end.py

```diff
@@ -80,7 +80,13 @@
     QueryError,
 )
 from ..file_store import FileStore
-from ..globals import BATCH_FORMAT_VERSION, HTTP_CLIENT_MAX_SIZE, RESERVED_STORAGE_GB_PER_CORE, complete_states
+from ..globals import (
+    BATCH_FORMAT_VERSION,
+    HTTP_CLIENT_MAX_SIZE,
+    RESERVED_STORAGE_GB_PER_CORE,
+    ROOT_JOB_GROUP_ID,
```

Comment:
I'm all for making `ROOT_JOB_GROUP_ID` a constant, but let's move it out of globals because to me that gives the impression that it is configurable or might change in the future. IIUC it is basically never a good idea to change this constant.

Reply:
Yep, this constant can never change once we choose it.